docs/contributing-guide.adoc

```diff
@@ -0,0 +1,88 @@
+== AeroGear Service Developer Guide
+
+Following guide describes architecture for Android SDK and instructions for developing new AeroGear Services.
+
+For instructions about using SDK in user applications please follow link:./getting-started.adoc[User Getting Started Guide]
+
+image:./images/diagram.svg[diagram,450,450,role="right"]
+
+=== SDK architecture
+
+SDK aggregates individual services SDK into one single manageable namespace. Services like AeroGear Sync, Push are build as separate SDK's that can be included into any existing or new Android applications. SDK using gradle for dependency resolution and distribution. Each SDK is represented as separate gradle module. Modules are used and are tested in `example` application that is also used for building and demo purposes. 
```

Comment:
we don't really have Sync and Push at the moment, so I think mention them here will cause some confusion. Better replace it with the services we currently have, like auth and metrics.